contracts/MaxLoopsLimitHelper.sol

```diff
@@ -23,6 +23,8 @@ abstract contract MaxLoopsLimitHelper {
      * @param limit Limit for the max loops can execute at a time
      */
     function _setMaxLoopsLimit(uint256 limit) internal {
+        require(limit > maxLoopsLimit, "Comptroller:: Invalid maxLoopsLimts");
```

Comment:
I think `:` is a bit cleaner. Some use `::` as a scoping operator (does it come from PHP?), but in this case it would rather be sth like `Comptroller::InvalidMaxLoopsLimit` so that "code" and plain text are not mixed

```suggestion
        require(limit > maxLoopsLimit, "Comptroller: Invalid maxLoopsLimit");
```

Reply:
Fixed, b0b1f5271c7435b2cb8ee5514c0b60f8c5023905